Sandbox/conan-recipe/conanfile.py

```diff
@@ -0,0 +1,159 @@
+import json
+import os
+import re
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+class NmosCppConan(ConanFile):
+    name = "nmos-cpp"
+    description = "An NMOS C++ Implementation"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/garethsb/nmos-cpp"
+    topics = ("NMOS")
+
+    settings = "os", "compiler", "build_type", "arch"
+    # no "shared" option support
+    # is anything else required to support "fPIC"?
+    options = {"fPIC": [True, False]}
```

Comment:
Nop! Conan will inject https://cmake.org/cmake/help/latest/variable/CMAKE_POSITION_INDEPENDENT_CODE.html

```suggestion
    options = {"fPIC": [True, False]}
```

Reply:
Thanks. Good to know. I don't intend to support the `shared` option right now though, is that OK?